### ISSUE TITLE
Failed auto-generating package requirements

### ISSUE THREAD

**Issue by KumoLiu** [allegroai/clearml#1361]
## Describe the bug
```
Failed auto-generating package requirements: too many values to unpack (expected 2)
Event reporting sub-process lost, switching to thread based reporting
```
related log: https://github.com/Project-MONAI/MONAI/actions/runs/12547070815/job/35058851817

## To reproduce
when running the unittest to initialize the clearml task, it will raise this error.
https://github.com/Project-MONAI/MONAI/blob/996e876e7542f683508aa04e74b97e284bbde72b/monai/handlers/clearml_handlers.py#L22

Not sure whether it related the change here: https://github.com/Project-MONAI/MONAI/pull/8260/files or related to the update of the clearml.

## Environment
* Server type (self hosted \ app.clear.ml)
* Clearml: 1.17.0
* Python Version: 3.9/ 3.10/ 3.11
* Linux

**Comment by eugen-ajechiloae-clearml** [comment]
Hi @KumoLiu ! Thank you for reporting.
Does this error also cause the tests to time out? Any idea how one could reproduce this locally?

**Comment by KumoLiu** [comment]
> Does this error also cause the tests to time out?

Based on the error message, yes.

> Any idea how one could reproduce this locally?

Actually I didn't find a good way to reproduce it locally. I tried run the unittest locally, it won't throw this error. Do you have any suggestions?

Thanks for the quick reply.

**Comment by eugen-ajechiloae-clearml** [comment]
@KumoLiu it likely has something to do with some packages. One thing worth trying is logging the packages installed on the unit test and installing exactly the same packages locally in a venv. Then running the test. If you can find out exactly what the packages are, could you please share them here?

**Comment by KumoLiu** [comment]
> @KumoLiu it likely has something to do with some packages. One thing worth trying is logging the packages installed on the unit test and installing exactly the same packages locally in a venv. Then running the test. If you can find out exactly what the packages are, could you please share them here?

Yes, I tried with the same packages and base container locally, can't reproduce. Here are all the packages.
<details><summary>packages</summary>
<p>
Package                            Version
---------------------------------- ---------------
absl-py                            2.1.0
alembic                            1.14.0
asciitree                          0.3.3
astor                              0.8.1
attrs                              24.3.0
beautifulsoup4                     4.12.3
black                              24.10.0
blinker                            1.9.0
cachetools                         5.5.0
certifi                            2024.12.14
cfgv                               3.4.0
charset-normalizer                 3.4.1
clearml                            1.17.0
click                              8.1.8
cloudpickle                        3.1.0
colorama                           0.4.6
coloredlogs                        15.0.1
colorlog                           6.9.0
contourpy                          1.3.0
coverage                           7.6.10
cucim-cu12                         24.8.0
cupy-cuda12x                       13.3.0
cycler                             0.12.1
databricks-sdk                     0.40.0
Deprecated                         1.2.15
distlib                            0.3.9
docker                             7.1.0
einops                             0.8.0
fasteners                          0.[19](https://github.com/Project-MONAI/MONAI/actions/runs/12547070815/job/35058851817#step:7:20)
fastrlock                          0.8.3
filelock                           3.11.0
fire                               0.7.0
flake8                             7.1.1
flake8-bugbear                     24.2.6
flake8-comprehensions              3.16.0
Flask                              3.1.0
flatbuffers                        24.12.23
fonttools                          4.55.3
fsspec                             [20](https://github.com/Project-MONAI/MONAI/actions/runs/12547070815/job/35058851817#step:7:21)24.12.0
furl                               2.1.3
gdown                              5.2.0
gitdb                              4.0.12
GitPython                          3.1.44
google-auth                        2.37.0
graphene                           3.4.3
graphql-core                       3.2.5
graphql-relay                      3.2.0
greenlet                           3.1.1
grpcio                             1.68.1
gunicorn                           23.0.0
h5py                               3.12.1
huggingface-hub                    0.27.0
humanfriendly                      10.0
humanize                           4.11.0
identify                           2.6.4
idna                               3.10
imagecodecs                        2024.12.30
imageio                            2.36.1
immutabledict                      4.2.1
importlab                          0.8.1
importlib_metadata                 8.5.0
importlib_resources                6.4.5
isort                              5.13.2
itk                                5.4.0
itk-core                           5.4.0
itk-filtering                      5.4.0
itk-io                             5.4.0
itk-numerics                       5.4.0
itk-registration                   5.4.0
itk-segmentation                   5.4.0
itsdangerous                       2.2.0
Jinja2                             3.1.5
joblib                             1.4.2
json-tricks                        3.17.3
jsonschema                         4.23.0
jsonschema-specifications          2024.10.1
kiwisolver                         1.4.7
lazy_loader                        0.4
libcst                             1.5.1
lmdb                               1.5.1
lpips                              0.1.4
Mako                               1.3.8
Markdown                           3.7
markdown-it-py                     3.0.0
MarkupSafe                         3.0.2
matplotlib                         3.9.4
mccabe                             0.7.0
mdurl                              0.1.2
MetricsReloaded                    0.1.0
mlflow                             2.19.0
mlflow-skinny                      2.19.0
mpmath                             1.3.0
msgspec                            0.19.0
mypy                               1.11.2
mypy-extensions                    1.0.0
networkx                           3.1
nibabel                            5.3.2
ninja                              1.11.1.3
nni                                2.10.1
nodeenv                            1.9.1
numcodecs                          0.12.1
numpy                              1.26.4
nvidia-cublas-cu12                 12.4.5.8
nvidia-cuda-cupti-cu12             12.4.127
nvidia-cuda-nvrtc-cu12             12.4.127
nvidia-cuda-runtime-cu12           12.4.127
nvidia-cudnn-cu12                  9.1.0.70
nvidia-cufft-cu12                  11.2.1.3
nvidia-curand-cu12                 10.3.5.147
nvidia-cusolver-cu12               11.6.1.9
nvidia-cusparse-cu12               12.3.1.170
nvidia-ml-py                       12.560.30
nvidia-nccl-cu12                   2.[21](https://github.com/Project-MONAI/MONAI/actions/runs/12547070815/job/35058851817#step:7:22).5
nvidia-nvjitlink-cu12              12.4.127
nvidia-nvtx-cu12                   12.4.127
onnx                               1.17.0
onnx-graphsurgeon                  0.5.2
onnxruntime                        1.19.2
openslide-python                   1.4.1
opentelemetry-api                  1.29.0
opentelemetry-sdk                  1.29.0
opentelemetry-semantic-conventions 0.50b0
optuna                             4.1.0
orderedmultidict                   1.0.1
packaging                          24.2
pandas                             2.2.3
parameterized                      0.9.0
pathlib2                           2.3.7.post1
pathspec                           0.12.1
pep8-naming                        0.14.1
pillow                             11.0.0
pip                                24.3.1
platformdirs                       3.11.0
polygraphy                         0.49.9
pre_commit                         4.0.1
prettytable                        3.12.0
protobuf                           5.29.2
psutil                             6.1.1
pyamg                              5.2.1
pyarrow                            18.1.0
pyasn1                             0.6.1
pyasn1_modules                     0.4.1
pycnite                            2024.7.31
pycodestyle                        2.12.1
pydicom                            2.4.4
pydot                              3.0.3
pyflakes                           3.2.0
Pygments                           2.18.0
PyJWT                              2.9.0
pynrrd                             1.1.1
pyparsing                          3.2.1
PySocks                            1.7.1
python-dateutil                    2.9.0.post0
PythonWebHDFS                      0.2.3
pytorch-ignite                     0.4.11
pytype                             2024.9.13
pytz                               2024.2
PyYAML                             6.0.2
referencing                        0.35.1
regex                              2024.11.6
requests                           2.32.3
responses                          0.25.3
rich                               13.9.4
rpds-py                            0.[22](https://github.com/Project-MONAI/MONAI/actions/runs/12547070815/job/35058851817#step:7:23).3
rsa                                4.9
ruff                               0.8.4
safetensors                        0.4.5
schema                             0.7.7
scikit-image                       0.22.0
scikit-learn                       1.6.0
scipy                              1.13.1
segment-anything                   1.0
setuptools                         58.1.0
shellingham                        1.5.4
SimpleITK                          2.4.0
simplejson                         3.19.3
six                                1.17.0
smmap                              5.0.2
soupsieve                          2.6
SQLAlchemy                         2.0.36
sqlparse                           0.5.3
sympy                              1.13.1
tabulate                           0.9.0
tensorboard                        2.18.0
tensorboard-data-server            0.7.2
tensorboardX                       2.6.2.2
termcolor                          2.5.0
threadpoolctl                      3.5.0
tifffile                           2024.8.30
tokenizers                         0.19.1
toml                               0.10.2
tomli                              2.2.1
torch                              2.5.1
torchio                            0.20.3
torchvision                        0.20.1
tqdm                               4.67.1
transformers                       4.40.2
triton                             3.1.0
typeguard                          2.13.3
typer                              0.15.1
types-PyYAML                       6.0.12.20241[23](https://github.com/Project-MONAI/MONAI/actions/runs/12547070815/job/35058851817#step:7:24)0
types-setuptools                   75.6.0.20[24](https://github.com/Project-MONAI/MONAI/actions/runs/12547070815/job/35058851817#step:7:25)1223
typing_extensions                  4.12.2
tzdata                             2024.2
urllib3                            2.3.0
virtualenv                         20.23.0
wcwidth                            0.2.13
websockets                         14.1
Werkzeug                           3.1.3
wheel                              0.45.1
wrapt                              1.17.0
zarr                               2.18.2
zipp                               3.21.0
2.5.1+cu124

</p>
</details>
